server/odc-service/src/main/java/com/oceanbase/odc/service/connection/model/ConnectionConfig.java

```diff
@@ -404,6 +404,7 @@ public int queryTimeoutSeconds() {
         return queryTimeoutSeconds;
     }
 
+    @Size(max = 8192, message = "Session init script is out of range [0,8192]")
```

Comment:
why limit script length to 8192

Reply:
1. the script length has to be limited
2. 8192 is determined by imaging, there is no more meanings.